cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -504,6 +508,62 @@ export function changePropagateFrames(frames: number): AnyAction {
     };
 }
 
+
+//To remove annotation objects present in given range of frames
+export function removeAnnotationsinRangeAsync(sessionInstance: any, startFrame: number, endFrame: number, force: boolean): ThunkAction{
+    return async (dispatch: ActionCreator<Dispatch>): Promise<void> => {
+        const { filters, showAllInterpolationTracks } = receiveAnnotationsParameters();
+        try {
+
+            for(let frame = startFrame; frame<=endFrame; frame++){
+                await dispatch(changeFrameAsync(frame));
```

Comment:
Do we really need to navigate each frame here? In this case annotation removing can fetch huge amount of time (getting frames from the server, decoding them, etc).